tests/flow/destructuring/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -165,9 +165,9 @@ obj_prop_var({}); // ok
 obj_prop_var({ p: {} }); // ok
 obj_prop_var({ p: { q: null } }); // ok, provides add'l lower bound
 
-function obj_rest(
-  { p: { q, ...o } = { q: 0, r: 0 } } = { p: { q: 0, r: "" } }
-) {
+function obj_rest({ p: { q, ...o } = { q: 0, r: 0 } } = {
+  p: { q: 0, r: "" }
+}) {
```

Comment:
this is weird but I think it's an edge case.. we could check if the right side is not another `AssignmentPattern`